atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/memory/InMemoryAtlasDbFactory.java

```diff
@@ -83,15 +87,22 @@ public TimestampService createTimestampService(KeyValueService rawKvs) {
     }
 
     public static SerializableTransactionManager createInMemoryTransactionManager(AtlasSchema schema) {
+        return createInMemoryTransactionManager(ImmutableSet.of(schema));
+    }
+
+    public static SerializableTransactionManager createInMemoryTransactionManager(Set<AtlasSchema> schemas) {
```

Comment:
(different method from the above one, because otherwise i think you put yourself at risk of binary backcompat issues?)